docs/guides/serverless-manage-resources.ipynb

```diff
@@ -22,22 +22,53 @@
   },
```

Comment:
I think line 20 should be `transpile_remote.py`. /guides/serverless-first-program#get-program-arguments also uses the wrong file name